### PR TITLE
Update doc of `SemanticsProperties.identifier`

### PR DESCRIPTION
**Pull request by mdebbar**
Updating the docs to reflect the change in https://github.com/flutter/engine/pull/53278.

`Semantics(identifier: '...')` can now be used on web to facilitate testing and DOM lookup when it comes to semantics nodes.

### CODE REVIEW

File: packages/flutter/lib/src/semantics/semantics.dart [hunk 1]
```diff
@@ -1199,6 +1199,9 @@ class SemanticsProperties extends DiagnosticableTree {
   /// It'll be appear in accessibility hierarchy as `resource-id`.
   ///
   /// On iOS, this will set `UIAccessibilityElement.accessibilityIdentifier`.
+  ///
+  /// On web, this will set a `semantics-identifier` attribute on the DOM element
```
Comment (chunhtai) [inline packages/flutter/lib/src/semantics/semantics.dart:1203]: does this have special meaning in native dom?

**Review by chunhtai — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/semantics/semantics.dart (hunk 1 above)
Comment (mdebbar) [reply packages/flutter/lib/src/semantics/semantics.dart:1203]: Nope. It's just a name that I chose.

File: packages/flutter/lib/src/semantics/semantics.dart [hunk 2]
```diff
@@ -1199,6 +1199,9 @@ class SemanticsProperties extends DiagnosticableTree {
   /// It'll be appear in accessibility hierarchy as `resource-id`.
   ///
   /// On iOS, this will set `UIAccessibilityElement.accessibilityIdentifier`.
+  ///
+  /// On web, this will set a `flt-semantics-identifier` attribute on the DOM element
+  /// that corresponds to the semantics node.
```
Comment (goderbauer) [inline packages/flutter/lib/src/semantics/semantics.dart:1204]: Removing extra white space:

```suggestion
  /// that corresponds to the semantics node.
```

**Review by yjbanov — APPROVED** [review]

**Pull request merged**